Code/GraphMol/ForceFieldHelpers/ANI/ANIForceFieldHelperTest.cpp

```diff
@@ -21,13 +21,15 @@
 #include <GraphMol/Descriptors/AtomicEnvironmentVector.h>
 #include <Eigen/Dense>
 #include <GraphMol/ForceFieldHelpers/ANI/Builder.h>
+#include "ANI.h"
 
 using namespace RDKit;
 
 TEST_CASE("Check ANI Force Field builder") {
```

Comment:
A number of these tests allocate memory but don't free it.
We try to ensure that we can use the tests to detect memory leaks, so it's important that the tests themselves don't leak.
Please either use unique_pointers for everything or be super careful about deleting your pointers